app/views/admin/loans/questionnaires/_breakeven_table_question.html.slim

```diff
@@ -25,6 +25,21 @@
         = render 'admin/loans/questionnaires/breakeven_product_row', question: question
 
   .table-container
+    h4.title = t("loan.breakeven.rampup")
```

Comment:
Used the same table style layout as the rest of the subform to keep consistency. Not sure if it would work better as normal form controls.